meta/saisanitycheck.c

```diff
@@ -372,7 +372,7 @@ void check_attr_by_object_type()
             sai_object_type_t current = ot[index]->objecttype;
 
             META_ASSERT_TRUE(current == i, "object type must be equal on object type list");
-            META_ASSERT_TRUE(index < 200, "object defines > 200 attributes, metadata bug?");
+            META_ASSERT_TRUE(index < 300, "object defines > 300 attributes, metadata bug?");
```

Comment:
how come this is triggered? i dont see any @brief tag to be longer than 200 chars ?